src/process_utility.c

```diff
@@ -2020,7 +2021,7 @@ process_rename_index(ProcessUtilityArgs *args, Cache *hcache, Oid relid, RenameS
 		Chunk *chunk = ts_chunk_get_by_relid(tablerelid, false);
 
 		if (NULL != chunk)
-			ts_chunk_index_rename(chunk, relid, stmt->newname);
+			ts_chunk_index_rename(chunk, indexname, stmt->newname);
```

Comment:
curious why we don't have coverage for this code path

Reply:
@fabriziomello , IIUC, this is not a problem with this PR? should we address it separately?